style.css

```diff
@@ -0,0 +1,266 @@
+.body {
+margin: 0;
+}
+
+.header {
+display: flex;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5906702)